cursorless-talon/src/modifiers/containing_scope.py

```diff
@@ -13,6 +13,7 @@
     "arg": "argumentOrParameter",
     "attribute": "attribute",
     "call": "functionCall",
+    "callee": "functionCallee",
```

Comment:
This is the first step of #645.  What do we want to use for the identifier and default spoken form?  I'd lean towards

```
    "callee": "functionCallFunction",
```

@AndrewDant feel free to implement this scope type for other languages as part of closing #645.  You can either wait for this PR to merge or just implement the couple relevant lines; the actual definition is fairly simple

cc/ @AndreasArvidsson @wenkokke @AndrewDant

Reply:
I'm not a huge fan of functionCallFunction. Not intuitively distinct from "call" and "functionCall", it could cause some confusion for someone trying to learn the code or debug some json.

Reply:
I've come around on callee though.

Reply:
Maybe `functionCallCallee`?